eland/operations.py

```diff
@@ -293,11 +303,18 @@ def _metric_aggs(
 
                 # If we have multiple 'extended_stats' etc. here we simply NOOP on 2nd call
                 if isinstance(es_agg, tuple):
-                    body.metric_aggs(
-                        f"{es_agg[0]}_{field.es_field_name}",
-                        es_agg[0],
-                        field.aggregatable_es_field_name,
-                    )
+                    if es_agg[0] == "percentiles":
+                        body.percentile_agg(
```

Comment:
It'd probably be good to use keyword arguments for these function calls to better illustrate what is going where.